src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int POSITION_OF_DATE = 0;
+    private static final int POSITION_OF_NAME = 1;
+    private static final int POSITION_OF_HOURS = 2;
+    private static final int POSITION_OF_MONEY_PER_HOUR = 3;
+    private static final String SEPARATOR = " ";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int moneyForPeriod;
+        StringBuilder builder = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo);;
```

Comment:
Two `;` at the end of line. Also `"Report for period "` and `" - "` should be constant Strings.